variables.tf

```diff
@@ -128,4 +128,53 @@ variable "s3_tag" {
 #         "us-east-1" = "ami-061ac2e015473fbe2"
 #         "us-east-2" = "ami-1234"
 #     }
-# }
\ No newline at end of file
+# }
+
+variable "db_allocated_storage" {
+  type = number
+  default = 10
+}
+
+variable "db_engine" {
+  default = "postgres"
+}
+
+variable "db_engine_version" {
+  default = "10.15"
+}
+
+variable "db_instance_class" {
+  default = "db.t2.micro"
+}
+
+variable "db_name" {
+  default = "ziyotek"
+}
+
+variable "db_username" {
+  default = "ziyotekuser"
+}
+
+variable "db_password" {
+  default = "ziyotek12345"
+}
+
+variable "db_skip_final_snapshot" {
+  default = true
+}
+

```

Comment:
that wont work here, only hard coded values are accepted. arguments cannot be referenced in vars